crates/compiler/mono/src/ir.rs

```diff
@@ -4938,43 +4939,56 @@ pub fn with_hole<'a>(
                 Err(_) => return runtime_error(env, "Can't update record with improper layout"),
             };
 
-            let mut field_layouts = Vec::with_capacity_in(sorted_fields.len(), env.arena);
+            let single_struct_field = sorted_fields.len() == 1;
```

Comment:
would `single_field_struct`? not read nicer?

Reply:
👍